### PR TITLE
Implement unit class for quarantine status

### LINKED ISSUE #34: Implement unit class for quarantined status
A person can either be quarantined or not quarantined by Covid-19. The handling of this property of a person requires a new unit class to be created. The quarantined status is independent of the infected status.

### PR DESCRIPTION
**Pull request by hopinxian**
This branch adds a new unit class in the model.person package. This new class encapsulates the property of the quarantine status of a person.

Close #34

### CODE REVIEW

**Review by hhdqirui — APPROVED** [review]
All look good to me.

File: src/test/data/JsonSerializableAddressBookTest/invalidPersonAddressBook.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "name": "Hans Muster",
     "phone": "9482424",
     "email": "invalid@email!3e",
-    "address": "4th street"
+    "address": "4th street",
+    "quarantineStatus": "tru"
```
Comment (KohHanMing) [inline src/test/data/JsonSerializableAddressBookTest/invalidPersonAddressBook.json:7]: Minor spelling error "true"

**Review by KohHanMing — APPROVED** [review]
Other than the spelling error, everything else looks good :)

File: src/test/data/JsonSerializableAddressBookTest/invalidPersonAddressBook.json (hunk 1 above)
Comment (hopinxian) [reply src/test/data/JsonSerializableAddressBookTest/invalidPersonAddressBook.json:7]: Spelling error is intentional for the invalid person.

**Pull request merged**